src/utils/startup.js

```diff
@@ -26,6 +27,19 @@ async function checkLogin({ to, from, next }) {
     return await store.dispatch('users/getProfile')
   } catch (e) {
     Vue.$log.error(e)
+    // remove currentOrg: System org item
+    for (let i = 0; i < localStorage.length; i++) {
+      const key = localStorage.key(i)
+      if (!key.startsWith('currentOrg:')) {
+        continue
+      }
+      let value = localStorage.getItem(key)
+      value = JSON.parse(value)
+      if (!value.is_system) {
+        continue
+      }
+      localStorage.removeItem(key)
+    }
     const status = e.response.status
     if (store.getters.currentOrg.autoEnter) {
       await store.dispatch('users/setCurrentOrg', store.getters.preOrg)
```

Comment:
There is an issue with the use of `localStorage`. It is not recommended to rely on local storage for sensitive operations such as user authentication. Consider using a secure cookie-based or JSON web token (JWT) instead. Additionally, removing items that are marked as "System" may affect other users as well if they have accessed those items directly.

I would recommend implementing a more secure solution like JWT based authentication tokens or cookies for future updates. This change will help improve security measures, which can mitigate potential risks associated with storing passwords.